app/server/src/controllers/appsController.ts

```diff
@@ -41,7 +41,9 @@ export class AppsController {
                     wodinVersion,
                     loadSessionId: sessionId || "",
                     shareNotFound: shareNotFound || "",
-                    mathjaxSrc: "https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js"
+                    mathjaxSrc: "https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js",
+                    i18n: wodinConfig.i18n ?? true, // if option not set then true by default
```

Comment:
should this be something like `enableI18n`? (and through things like the .hbs file)

Reply:
done